components/Button/Button.css

```diff
@@ -30,6 +33,7 @@
 }
 
```

Comment:
Since you are using same transition multiple times, you can assign a variable to it.

Reply:
Sure thanks